src/plugins/discover/public/application/main/utils/update_saved_search.ts

```diff
@@ -54,6 +54,9 @@ export function updateSavedSearch({
       .setField('query', state.query ?? undefined)
       .setField('filter', [...globalFilters, ...appFilters]);
   }
+  if (savedSearch.timeRestore) {
+    savedSearch.timeRange = services.timefilter.getTime();
+  }
```

Comment:
```suggestion
```

It should already work based on the code at the end of the function. Doesn't it?

Reply:
You're right, missed that, thanks!